parameters/world_modeling/world_model.yaml

```diff
@@ -36,7 +36,7 @@ plugins:
   parameters:
       constraint_service:
           default_offset: 0.7
-          room_offset: 0.3
+          room_offset: -0.3
```

Comment:
Why did this change?

Reply:
It is an offset. So positive would make the room bigger. Negative the room smaller. Which is the desired behaviour.